flocker/acceptance/obsolete/test_containers.py

```diff
@@ -123,6 +123,7 @@ def checked(_):
 
     @flaky(u'FLOC-2488')
     @require_moving_backend
+    @run_test_with(async_runner(timeout=timedelta(minutes=6)))
```

Comment:
It might.... I guess I'll remove the flaky decorator for now and see if we regret it later.